File: PSDF_BSS.Reports/Models/ModelBase.cs
using System;

namespace DataLayer.Models
{
    public class ModelBase
    {
        public ModelBase()
        {
        }

        public ModelBase(bool InActive)
        {
            this.InActive = InActive;
        }

        public int CreatedUserID { get; set; }
        public int CurUserID { get; set; }
        public int? ModifiedUserID { get; set; }
        public DateTime? CreatedDate { get; set; }
        public DateTime? ModifiedDate { get; set; }

        public bool? InActive { get; set; }
    }
}

File: Frontend/ClientApp/src/app/shared/ModelBase.ts
export class ModelBase {
  CreatedUserID: number;
  ModifiedUserID: number;
  CreatedDate: string;
  ModifiedDate: string;
  InActive: boolean;
}
export interface PagingModel {
  PageNo: number;
  PageSize: number;
  TotalCount: number;
  SortOrder: string;
  Sortdir: string;
  SearchColumn: string;
  SearchValue: string;
}
